packages/jest-reporters/src/coverage_reporter.ts

```diff
@@ -4,48 +4,45 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @flow
  */
 
-import type {
-  AggregatedResult,
-  CoverageMap,
-  FileCoverage,
-  CoverageSummary,
-  TestResult,
-} from 'types/TestResult';
-import typeof {worker} from './coverage_worker';
-
-import type {GlobalConfig, Path} from 'types/Config';
-import type {Context} from 'types/Context';
-import type {Test} from 'types/TestRunner';
+import path from 'path';
+import {TestResult, Config} from '@jest/types';
 
 import {clearLine, isInteractive} from 'jest-util';
+// @ts-ignore Apparently there aren't any types for this package?
 import {createReporter} from 'istanbul-api';
 import chalk from 'chalk';
-import istanbulCoverage from 'istanbul-lib-coverage';
-import libSourceMaps from 'istanbul-lib-source-maps';
+import istanbulCoverage, {
+  CoverageMap,
+  FileCoverage,
+  Totals,
+} from 'istanbul-lib-coverage';
+import libSourceMaps, {MapStore} from 'istanbul-lib-source-maps';
 import Worker from 'jest-worker';
-import BaseReporter from './base_reporter';
-import path from 'path';
 import glob from 'glob';
+import BaseReporter from './base_reporter';
+import {
+  Context,
+  Test,
+  CoverageSummary,
```

Comment:
I tried to create `CoverageSummary` here but it conflicts with istanbul's own `CoverageSummary` which targets an old version without the `merge` function.